samples/highcharts/demo/parallel-coordinates-polar/demo.html

```diff
@@ -0,0 +1,14 @@
+<script src="https://code.highcharts.com/highcharts.js"></script>
+<script src="https://code.highcharts.com/highcharts-more.js"></script>
+<script src="https://code.highcharts.com/modules/parallel-coordinates.js"></script>
+<script src="https://code.highcharts.com/modules/exporting.js"></script>
+<script src="https://code.highcharts.com/modules/accessibility.js"></script>
```

Comment:
There are styles declared for the data table, but currently, the `export-data` module is not loaded. I assume it should be loaded 😁